### ISSUE TITLE
It's said "Animation contains intersect masks", which does not exist (android)

### ISSUE THREAD

**Issue by HYlee-designer** [airbnb/lottie-android#784]
<img width="1133" alt="2018-05-24 2 59 54" src="https://user-images.githubusercontent.com/21138364/40467029-28fae266-5f63-11e8-9be4-d445cc616152.png">

So I'm currently making the loading images for the new project.
But my developer told me the warning message saying "W/LOTTIE: Animation contains intersect masks. They are not supported but will be treated like add masks." keeps on appearing, and I found the information that android OS doesn't support the intersect masks unlike iOS so I try to avoid the "intersect masks" as possible by separating the same logo layers with masks into 2 like the screen capture above (I changed the color of the eclipse mask to make it recognizable). 
So, as far as I can see, there is no intersect mask here, as it looks the same but each of the masks are divided.
BUT THAT WARNING MESSAGE STILL APPPEARS!
Is there no way to work with it besides: 
draw them frame by frame back again from the illustrator->get the shapes at the After Effects->attach the shapes frame by frames again and so on, just for the loading icon ??

The loading icon should be like this:
![loading](https://user-images.githubusercontent.com/21138364/40467367-9900de98-5f64-11e8-91a8-c38d421bf413.gif)

Thank you.

**Comment by vfurkana** [comment]
We're having the same issue.
This is also causing performance issues. I tried dozens of animations and the ones with "intersect masks" message have performance problems while others worked smoothly. 
I asked our designer to remove the mask elements and messages were gone. Then we tried it with only 1 mask and I received messages again. So it's not related to having more than one masks.
Animations are working fine on iOs and Web.
